src/test/java/org/jenkinsci/plugins/matrixauth/AuthorizationContainerDescriptorTest.java

```diff
@@ -21,7 +21,7 @@
     @Rule
     public JenkinsRule r = new JenkinsRule(); // Needed to check the jenkins version
 
-    private Permission FAKE = new Permission(Item.PERMISSIONS, "Fake", new TestLocalizable(), Item.BUILD, PermissionScope.ITEM);
+    private Permission TEST_PERMISSION = new Permission(Item.PERMISSIONS, "Test", new Localizable(new ResourceBundleHolder(AuthorizationContainerDescriptorTest.class), "Test"), Item.BUILD, PermissionScope.ITEM);
```

Comment:
I tried to do this but I didn't find the way. I saw [here](https://wiki.jenkins.io/display/JENKINS/Internationalization), that _Jenkins uses localizer to generate classes for accessing message resources in a type-safe way. For each src/main/resources/**/Messages.properties, a Messages class is generated. See the referenced page for how this class looks like. If your IDE fails to find these classes, manually add "target/generated-sources/localizer" directory to your source root._, so I thought that a properties won't be loaded as test resource. I had a look at the [maven-localizer-plugin](https://github.com/kohsuke/localizer) that is configured in parent-pom as well and it make me think my first research was ok.
Good to know. Thanks!

Reply:
Hacky approach, obviously: We're bypassing the code generation here.